src/components/idleUsers/card/card.module.scss

```diff
@@ -8,16 +11,23 @@
   flex-wrap: wrap;
   margin: 2rem;
   padding: 2rem;
-  cursor: pointer;
   align-items: center;
 
   img {
     border-radius: 50%;
   }
 
   .name {
+    cursor: pointer;
```

Comment:
What if I am hovering on the image, it should be a cursor pointer rather than just keeping it on the name.

Reply:
I have made only name clickable not the whole card